### PR TITLE
Xenobiology Revitalisations

### PR DESCRIPTION
**Pull request by sadkermit**
Overall just adds some more stuff to Xenobiology. Gives them some more tools to help Medical/Hydroponics/Xenobotany too so they're actually seen outside of their lab.

**Slime Re-additions**
- Re-adds Cerulean slimes which create Extract Enhancer. They're unchanged from however they were last implemented.
- Re-adds Sepia slimes. They are different to how they were previously implemented, and instead open material portals which generate organic materials. Injecting water creates a material portal for wood/cardboard; injecting blood creates a material portal for leather/hide.
- Re-adds Pyrite slimes. They are different to how they were previously implemented, and instead can be used to upgrade some chemicals. It'll have Xenobiology interacting with Medical a bit more upgrading chemicals when there's no pharmacist, as well as Xenobotany next door to upgrade fertitlisers/mutagens.

**Pyrite Transmutations**
Mentioned above, Pyrite slime extracts can upgrade chemicals. The following have been added:
- 5 units of Cryoxadone --> 5 units of Clonexadone
- 5 units of Inaprovaline --> 10 units of Coagzolug
- 5 units of Kelotane --> 10 units of Dermaline
- 5 units of Bicaridine --> 10 units of Butazoline
- 5 units of Dexalin --> 10 units of Dexalin Plus
- 5 units of Mercury --> 1 unit of Dextrotoxin
- 5 units of EZ Nutrient --> 10 units of Diethylamine
- 5 units of Radium --> 10 units of Unstable Mutagen
- 5 units of Sugar --> 10 units of Hyperzine

**Phoron Costs**
Currently, the majority of extracts just require _a_ unit of phoron to do their effect. Now, some of the more powerful slime extracts require more than a single unit. 
- Green slime teleport ability requires 5u of Phoron.
- Gold slime creature spawn ability requires 10u of Phoron.
- Adamantine slime golem spawn ability requires 5u of Phoron.

**New Abilities to Prexisting Slimes**
- Green slime extracts can be infused with 10u silver + 10u carbon to create a Bluespace Crystal.
- Red slime extracts can be injected with 10u phoron to create Red Nightshade. Formerly, this created Glycerol - sugar is now used to create Glycerol.
- Pink slime extracts can be injected with water to create Paxazide.
- Oil slime extracts can be injected with water to create a material portal which spawns plastic and glass.

**Slime Jelly Tweaks**
- Slime jelly no longer nukes the liver with 100-300 toxins. Instead it deals genetic damage, complimenting the primary means of damage slime deal.
- Slime jelly's healing properties have been nerfed, though remain fairly strong. It can probably be used in desperate cases relatively safely. 

**Misc.**
- Renames the paths for the docility serum and extract enhancer from `slimepotion2` and `slimesteroid2` to `docility_serum` and `extract_enhancer`. Just makes it easier to search for them.

### CODE REVIEW

**Comment by sadkermit** [comment]
!review

**Comment by sadkermit** [comment]
Probably needs a wiki update tag too. idk if I can add that myself

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]
Remove the slime ghost role. We've seen how these end up before and it's never a good idea (remember random spawn golem runes?).

File: code/modules/mob/living/carbon/slime/items.dm [hunk 1]
```diff
@@ -123,36 +123,37 @@
 
 ////Pet Slime Creation///
 
-/obj/item/slimepotion
-	name = "docility potion"
-	desc = "A potent chemical mix that will nullify a slime's powers, causing it to become docile and tame."
+/obj/item/docility_serum
+	name = "docility serum"
+	desc = "A potent chemical mix that will nullify a slime's powers, causing it to become docile and tame. This one is meant for baby slimes"
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/carbon/slime/items.dm:128]:
```suggestion
	desc = "A potent chemical mix that will nullify a slime's powers, causing it to become docile and tame. This one is meant for baby slimes."
```

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Llywelwyn — APPROVED** [review]
code looks good. science changes are nice to see.

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**